docs/Actions.md

```diff
@@ -323,7 +323,9 @@ import_certificate certificate_path: "certs/dist.p12", certificate_password: ENV
 Enables the use of the `xcodebuild` tool within fastlane to perform xcode tasks
 such as; archive, build, clean, test, export & more.
 
-You should check out the [code signing guide](https://github.com/fastlane/fastlane/blob/master/docs/CodeSigning.md).
+**Note**: It is recommended to use [gym](https://github.com/fastlane/gym) for building your ipa file and [scan](https://github.com/fastlane/scan) for testing your app.
```

Comment:
The wording should be a little stronger here like:
"It is recommended to use gym for building your ipa file and scan for testing your app, instead of using xcodebuild directly."
or
"xcodebuild is a complex command, so it is recommended instead to use gym for building your ipa file and scan for testing your app."